core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -30,6 +30,7 @@ BadFormat=not in format ''{0}''
 BetweenWithoutAnd=BETWEEN operator has no terminating AND
 GeometryDisabled=Geo-spatial extensions and the GEOMETRY data type are not enabled
 IllegalIntervalLiteral=Illegal INTERVAL literal {0}; at {1}
+UnsupportedSnowflakeIntervalLiteral=Unsupported Snowflake style INTERVAL literal {0}; at {1}. Commas are not supported yet in interval literals
```

Comment:
How come we have to specify the format string here when we already had to in `core/src/main/java/org/apache/calcite/runtime/CalciteResource.java`? Assuming something in Calcite requires us to, but wanted to ask just in case.

Reply:
This is for a calcite test that verifies the resource exists. It possible this can be automatically generated but I don't know how.